website/content/docs/auth/userpass.mdx

```diff
@@ -65,27 +66,27 @@ management tool.
 
 1. Enable the userpass auth method:
 
-   ```shell-session
-   $ vault auth enable userpass
-   ```
+  ```shell-session
+  $ vault auth enable userpass
+  ```
 
-   This enables the userpass auth method at `auth/userpass`. To enable it at a different path, use the `-path` flag:
+  This enables the userpass auth method at `auth/userpass`. To enable it at a different path, use the `-path` flag:
```

Comment:
Suggest simplifying the instructions by adding this to the text before the example command as:

Enable the userpass auth method at the default `auth/userpass` path. You can choose to enable the auth method at a different path with the `-path` flag: